src/pages/guides/rpcs-and-das.md

```diff
@@ -0,0 +1,74 @@
+created: '06-16-2024'
+updated: '06-21-2024'
+---
+
+## Roles of an RPC on the Solana Blockchain
+Remote Procedure Calls (RPCs) are a crucial part of the Solana blockchain infrastructure. They serve as the bridge between users (or applications) and blockchain, facilitating interactions and data retrieval.
+
+#### Key Roles of an RPC
+1. **Facilitating Network Communication**
+RPC servers handle requests from clients (users or applications) and interact with the blockchain to fulfill those requests. They provide a standardized way for external entities to communicate with the blockchain without requiring them to run a full node.
+
+2. **Submitting Transactions**
+RPCs enable clients to submit transactions to the Solana blockchain. When a user wants to perform an action on the blockchain, such as transferring tokens or invoking a smart contract, the transaction is sent to an RPC server, which then propagates it to the network for processing and inclusion in a block.
+
+3. **Retrieving Blockchain Data**
+RPC servers allow clients to ask the blockchain for various types of data, including:
+- **Account Information**: details about a specific account, such as balance, token holdings, and other metadata.
+- **Transaction History**: historical transactions associated with an account or a specific transaction signature.
+- **Block Information**: details about specific blocks, including block height, block hash, and transactions included in the block.
+- **Program Logs**: Access logs and output from executed programs (smart contracts).
+
+4. **Monitoring Network Status**
+RPCs provide endpoints to check the status of the network and nodes, such as:
+- **Node Health**: Determine if a node is online and functioning correctly.
+- **Network Latency**: Measure the time it takes for requests to be processed and responses to be received.
+- **Synchronization Status**: Check if a node is synchronized with the rest of the network.
+
+5. **Supporting Development and Debugging**
+RPC endpoints are essential tools for developers building on Solana. They provide functionalities to:
+- **Simulate Transactions**: simulate transactions to see their potential effects before submitting them to the network.
+- **Fetch Program Accounts**: retrieve all accounts associated with a specific program, which is useful for managing program state.
+- **Get Logs**: detailed logs from transactions and programs to debug and optimize their applications.
+
+### Example RPC Endpoints
+Here are some common RPC endpoints and their functionalities:
+- **getBalance**: Retrieves the balance of a specified account.
+- **sendTransaction**: Submits a transaction to the network.
+- **getTransaction**: Fetches details about a specific transaction using its signature.
+- **getBlock**: Retrieves information about a specific block by its slot number.
+- **simulateTransaction**: Simulates a transaction to predict its outcome without executing it on the chain.
+
```

Comment:
should we also point to our RPC overview page where we have the overview which ones are supporting DAS?